verticapy/tests/vModel/test_lasso.py

```diff
@@ -147,10 +150,15 @@ def test_to_sklearn(self, model):
         prediction = model.cursor.fetchone()[0]
         assert prediction == pytest.approx(md.predict([[3.0, 11.0, 93.0]])[0][0])
 
-    @pytest.mark.skip(reason="shap doesn't want to work on python3.6")
-    def test_shapExplainer(self, model):
-        explainer = model.shapExplainer()
-        assert explainer.expected_value[0] == pytest.approx(5.81837771)
+    try:
+        import shap
+
+        def test_shapExplainer(self, model):
+            explainer = model.shapExplainer()
+            assert explainer.expected_value[0] == pytest.approx(5.81837771)
+
+    except:
+        pass
```

Comment:
It is important to fix this; otherwise, how can we know if the test is really performed.
Please use `pytest.skip(reason)` instead of `pass`.